internal/osutil/file.go

```diff
@@ -36,3 +39,92 @@ func WriteFileWithPermission(path string, data []byte, perm fs.FileMode, overwri
 	}
 	return file.Close()
 }
+
+// Copy the src file to dst. Existing file will be overwritten.
+func Copy(src, dst string) (int64, error) {
+	sourceFileStat, err := os.Stat(src)
+	if err != nil {
+		return 0, err
+	}
+
+	if !sourceFileStat.Mode().IsRegular() {
+		return 0, fmt.Errorf("%s is not a regular file", src)
+	}
```

Comment:
We can't copy from a symbolic link to the store?

Reply:
According to the [spce](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-store):
Symlinks are not supported for the named store directories or certificate files. Implementation MUST validate that the named store directory or certificate files are not symlinks, and fail if it does not meet this condition.

Reply:
I thought only the target cannot be a symbolic link, but the source file can be a symbolic link to another file?